.github/workflows/auto-reviewer.yml

```diff
@@ -1,12 +1,11 @@
 name: 'Reviewer lottery'
 on:
   pull_request:
-    types: [opened, ready_for_review, reopened]
-    tags-ignore:
-      - pair-programmed
+    types: [labeled]
 
 jobs:
-  test:
+  assign-reviewer:
+    if: ${{ github.event.label.name != 'pair-programmed' }}
```

Comment:
Shouldn't it be `labels` or so, according to the [event payload example](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-example-32)? PRs can have more than one label.